### PR TITLE
IA-1649: update npm version dockerfile

### PR DESCRIPTION
**Pull request by codeKameleon**
Explain what problem this PR is resolving

the docker build config didn't install the right version of npm

## Self proof reading checklist

- [ ] Did I use eslint and black formatters
- [ ] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [ ] My migrations file are included
- [ ] Are there enough tests
## Changes

- install npm 8.5 instead of npm 7
- remove node modules and forcing cache clean

### CODE REVIEW

File: docker/bundle/Dockerfile [hunk 1]
```diff
@@ -18,6 +18,10 @@ COPY package-lock.json .
 
 
 RUN npm install -g npm@8.5
+
+RUN rm -rf node_modules
```
Comment (quang-le) [inline docker/bundle/Dockerfile:22]: I don't understand why we do this. `npm ci` deletes the node_modules anyway. cf [https://docs.npmjs.com/cli/v6/commands/npm-ci](https://docs.npmjs.com/cli/v6/commands/npm-ci)

File: docker/bundle/Dockerfile [hunk 2]
```diff
@@ -18,6 +18,10 @@ COPY package-lock.json .
...
+RUN npm cache clean --force
+RUN npm cache verify
```
Comment (quang-le) [inline docker/bundle/Dockerfile:24]: It seems to me these commands are mutually exclusive: if we clean the cache there's  nothing to verify, and if we want to "heal" the cache with `verify` there's no point in deleting it

**Review by quang-le — COMMENTED** [review]
Not sure I want to block over this, but it may save use some build time to remove useless commands

File: docker/bundle/Dockerfile (hunk 1 above)
Comment (codeKameleon) [reply docker/bundle/Dockerfile:22]: Indeed, i think we can remove it @beygorghor  ?

File: docker/bundle/Dockerfile (hunk 2 above)
Comment (codeKameleon) [reply docker/bundle/Dockerfile:24]: yes ok, we remove the command `npm cache verify` ?

**Comment by olethanh** [comment]
err for the modification on the bundle/Docker that feels like a code Smell, can someone justify theses modification and why they are here ?

**Pull request merged**